.golangci.yml

```diff
@@ -13,6 +13,7 @@ linters:
     - bodyclose
     - exportloopref
     - forbidigo
+    - staticcheck
```

Comment:
This doesn't seem to be doing anything at all.

Even when adding this line running `make lint` locally it does not detect any new errors.

Steps to reproduce:
1. Checkout `main` branch
1. Run `make lint`. Observe how no errors are detected
1. Add the `staticcheck` line as done in this PR
1. Run `make lint`. Observe how no errors are detected

If that line really did something running the previous steps would detect errors, as in the `main` branch the changes of this PR are not available.

Reply:
I believe the reason is this: https://github.com/golangci/golangci-lint/discussions/1989

Instead of `staticcheck`, we should enable `stylecheck`.
Then, this configuration limits stylecheck to check only error capitalization:
```
  stylecheck:
    checks: ["ST1005"]
```
(see https://golangci-lint.run/usage/linters/#stylecheck)

Reply:
Thanks @miguelsorianod @ziccardi ! Just pushed the suggested changes.